### ISSUE TITLE
Readme updates

### ISSUE THREAD

**Issue by tomallpress** [cgkineo/adapt-hint#28]
> `The hint button is displayed between the component display text (header) and the interaction (widget). For components without display text, the button is displayed after the interaction (widget).`

I think there may be an error in the 2nd sentence. Not sure it's quite accurate. i think it's suggesting that components without an interaction, the hint button displays at the bottom of the component (which it does, in testing) but the wording is saying "without display text" which feels a bit vague. So i suggest rewording the 2nd sentence so it reads "For components without an interactive element (e.g. a Text component), the button is displayed after the instruction field."

**Comment by oliverfoster** [comment]
The "interaction" or "widget" is the section below the instruction text and before the end of the component. The text component does not have a widget, the slider, mcq, hotgraphic, graphic, accordion etc, all have a widget. The "display text" means the "header", which is the title, body and instruction text sections, all components apart from blank have a "header". Most components have a "header" and "widget". A header can be excluded by leaving the title, body and instruction blank.

It's very confusing that display text and interaction are used instead of header and widget.

> "For components without an interactive element (e.g. a Text component), the button is displayed after the instruction field."

I think it's more "For components with a header the button is displayed after the header. For components without a header the button is displayed after the widget.".

**Comment by tomallpress** [comment]
thanks @oliverfoster that makes sense. and your suggested re-word makes sense to me. Just for clarity could we also add the following (see in italics below) to your suggestion, as I'm not sure we're familiar with referring to the collective of title, body and instruction as the 'header':

"For components with a header _**(i.e. the display title, body and instruction fields)**_ the button is displayed after the header. For components without a header the button is displayed after the widget."

**Comment by kirsty-hames** [comment]
The Readme [demo.gif](https://github.com/cgkineo/adapt-hint/blob/master/demo.gif) could do with updating to reflect the Hint button position.
